src/DataViews/include/DataViews/AppInterface.h

```diff
@@ -5,11 +5,27 @@
 #ifndef DATA_VIEWS_APP_INTERFACE_H_
 #define DATA_VIEWS_APP_INTERFACE_H_
 
+#include <absl/types/span.h>
+#include <stdint.h>
+
+#include <string>
+
+#include "ClientData/ModuleData.h"
+#include "ClientData/ProcessData.h"
+#include "ClientModel/CaptureData.h"
+#include "OrbitBase/Future.h"
+#include "capture_data.pb.h"
+
 namespace orbit_data_views {
 
 class AppInterface {
  public:
   virtual ~AppInterface() noexcept = default;
+
+  virtual void SetClipboard(const std::string& contents) = 0;
+  [[nodiscard]] virtual std::string GetSaveFile(const std::string& extension) const = 0;
```

Comment:
what is savefile? also should it return std::filesystem::path?

Reply:
`GetSaveFile` opens a Qt file picker dialog and returns a file path. Yes it could probably return a std::filesystem::path, but this commit is merely defining an abstract interface for an already existing function. Improving on that is out of scope or do you see a need this to change right now that I missed?

Reply:
it is out of scope yes.